doc/assets/js/theme.js

```diff
@@ -26,65 +26,76 @@ function addNavBarBurgers(){
   }
 }
 
-function addAnchor(element) {
-  element.insertAdjacentHTML('beforeend',
-  `<a class="header-hash" href="#${element.id}" ariaLabel="Anchor">#</a>` )
-}
-
-function addAnchors() {
-  var headers = document.querySelectorAll(
-    '.docs-content h2[id], .docs-content h3[id], .docs-content h4[id], .docs-content h5[id], .docs-content h6[id]')
-    if (headers) {
-        headers.forEach(addAnchor)
-    }
-}
-
-function addTabs(){
-  const tabs = [...document.querySelectorAll('.tabs li')]
-  const tabContent = [...document.querySelectorAll('.tab-content section')]
-  const activeClass = 'is-active'
-
-  function init() {
-    if (window.sessionStorage.getItem('tabActive')) {
-      toggleTab(window.sessionStorage.getItem('tabActive'))
-    } else if (tabs && tabs[0]) {
-      let key = tabs[0].getAttribute('data-tab')
-      toggleTab(key)
+function initializeTabsAndAnchors () {
+  var anchorTable = {}
+
+  function createAnchorTable(element) {
+    const tabContainer = element.closest('.tab-content')
+    if(tabContainer) {
+      const tab = element.parentElement
+      const tabName = tab.getAttribute('data-content')
+      anchorTable[element.id] = tabName
     }
+  }
 
-    tabs.forEach(tab => {
-      tab.addEventListener('click', handleClick)
-    })
+  function addAnchor(element) {
+    element.insertAdjacentHTML('beforeend',
+    `<a class="header-hash" href="#${element.id}" ariaLabel="Anchor">#</a>` )
+  }
 
-    document.querySelectorAll('figure').forEach(fig => {
-      fig.addEventListener('click', event => {
-        basicLightbox.create(`<img src="${event.target.src}" alt="${event.target.alt}" />`).show()
-      })
-    })
+  function addAnchors() {
+    var headers = document.querySelectorAll(
+      '.docs-content h2[id], .docs-content h3[id], .docs-content h4[id], .docs-content h5[id], .docs-content h6[id]')
+      if (headers) {
+          headers.forEach(addAnchor)
+          headers.forEach(createAnchorTable)
+      }
   }
 
-  function handleClick(event) {
-    event.preventDefault()
-    let tab = event.currentTarget
-    let key = tab.getAttribute('data-tab')
+  function hashChanged() {
+    const hash = window.location.hash
+    const id = window.location.hash.substring(1)
+    const key = anchorTable[id]
     toggleTab(key)
+    document.getElementById(id).scrollIntoView()
+  }
+
+  function addOnHashChange () {
+    if ("onhashchange" in window) { // event supported?
```

Comment:
```suggestion
    if ('onhashchange' in window) { // event supported?
```

Please use quotes consistently.